atd-vzv/src/constants/nav.js

```diff
@@ -1,3 +1,7 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChartBar, faMap } from "@fortawesome/free-solid-svg-icons";
```

Comment:
these nav buttons look so much nicer with simple icons and new outlines. 👍

Reply:
I agree! These details from feedback snapped into place nicely.